pkg/scheduler/cache/cache.go

```diff
@@ -1497,9 +1499,13 @@ func (sc *SchedulerCache) RecordJobStatusEvent(job *schedulingapi.JobInfo, updat
 			if len(msg) == 0 {
 				msg = baseErrorMessage
 			}
-			if err := sc.taskUnschedulable(taskInfo, reason, msg, nominatedNodeName); err != nil {
-				klog.ErrorS(err, "Failed to update unschedulable task status", "task", klog.KRef(taskInfo.Namespace, taskInfo.Name),
-					"reason", reason, "message", msg)
+			ts, exist := schedulingutil.GetPodStatusLastSetCache(job.UID, taskInfo.UID)
```

Comment:
This part of code is used to reduce api requests sent to api-server? Need to split this as a separate commit, not to mit it into preemption performance improvement.